src/__tests__/ConnectionFromMongoCursor.spec.js

```diff
@@ -178,3 +178,41 @@ it('should return connection from mongo cursor using first 1 and last as null',
   expect(resultFirstPage.edges.length).toBe(1);
   expect(loader).toHaveBeenCalledTimes(1);
 });
+
+it('should not send the entire collection when before offset is 0', async () => {
+  await Promise.all(Array.from({ length: 50 }).map(createUser));
+
+  const cursor = UserModel.find();
+  const context = {};
+
+  const loader = jest.fn();
+  loader.mockReturnValue('user');
+
+  const args1 = {
+    before: offsetToCursor(0),
+  };
+
+  const args2 = {
+    before: offsetToCursor(0),
+    last: 9,
```

Comment:
you can't get 9 items before the cursor of index 0

the cursor should be at last 9

Reply:
the result should be 0 items in this case

Reply:
nice catch

Reply:
@sibelius True. I was trying to demonstrate that the fix you implied does not work as expected. When the `before` arg is `0`, even [when we set `last` to 10 by default](https://github.com/entria/graphql-mongoose-loader/pull/35/commits/f21a41ffd5757d33cf35109db43748a8ab1e113c#diff-53722dc92b9e827b5e93d11c4f7bd0a3R82), or pass anything; it will set the limit to `0` and send all the documents regardless.